app/controllers/concerns/sufia/file_sets_controller_behavior.rb

```diff
@@ -29,6 +29,7 @@ module FileSetsControllerBehavior
 
     def new
       @upload_set_id = ActiveFedora::Noid::Service.new.mint
+      @presenter = Sufia::WorkShowPresenter.new(::SolrDocument.new, nil)
```

Comment:
This presenter is used so that work_id isn't loaded from fedora, but is using solr instead?  Having the work presenter in the fileset controller seem incongruous.